src/scripts/main.js

```diff
@@ -1,3 +1,40 @@
 'use strict';
 
-// write code here
+const employees = document.querySelector('.employess');
+const employeesArray = [...employees.children];
+
+employeesArray.forEach(item => {
```

Comment:
What about this one?
<img width="839" alt="image" src="https://user-images.githubusercontent.com/32794096/232691096-49d1bb49-c428-4413-ac63-075100e1b624.png">